src/components/standalone/security/ips/IpsDisabledRules.vue

```diff
@@ -0,0 +1,221 @@
+      :title="t('standalone.ips.error_loading_disabled_rules')"
+      kind="error"
+    />
+    <NeSkeleton v-if="loading" :lines="10" />
+    <template v-else-if="rules.length > 0">
+      <div class="flex flex-col flex-wrap justify-between gap-4 md:flex-row">
+        <NeTextInput v-model="filter" :placeholder="t('common.filter')">
+          <template #prefix>
+            <FontAwesomeIcon :icon="faMagnifyingGlass" aria-hidden="true" class="h-4 w-4" />
+          </template>
+        </NeTextInput>
+        <NeButton v-if="rules.length > 0" kind="secondary" size="lg" @click="disablingRule = true">
+          <template #prefix>
+            <FontAwesomeIcon :icon="faXmarkCircle" aria-hidden="true" class="h-4 w-4" />
+          </template>
+          {{ t('standalone.ips.disable_rule') }}
+        </NeButton>
+      </div>
+      <NeTable
+        :ariaLabel="t('standalone.ips.rules_table')"
+        :skeleton-columns="7"
+        :skeleton-rows="5"
+        :sortDescending="sortDescending"
+        :sortKey="sortKey"
+        card-breakpoint="xl"
+      >
+        <NeTableHead>
+          <NeTableHeadCell column-key="description" sortable @sort="onSort">
+            {{ t('standalone.ips.rule_description') }}
+          </NeTableHeadCell>
+          <NeTableHeadCell column-key="id" sortable @sort="onSort">
+            {{ t('standalone.ips.rule_id') }}
+          </NeTableHeadCell>
+          <NeTableHeadCell>
+            <!-- no header for actions -->
+          </NeTableHeadCell>
+        </NeTableHead>
+        <NeTableBody>
+          <NeTableRow v-for="item in paginatedItems" :key="`${item.id}`">
+            <NeTableCell :data-label="t('standalone.ips.rule_description')">
+              {{ item.description }}
+            </NeTableCell>
+            <NeTableCell :data-label="t('standalone.ips.rule_id')">
+              {{ item.id }}
+            </NeTableCell>
+            <NeTableCell :data-label="t('common.actions')">
+              <div class="flex justify-end">
+                <NeDropdown :align-to-right="true" :items="dropDownActions(item)" />
+              </div>
+            </NeTableCell>
+          </NeTableRow>
+        </NeTableBody>
+        <template #paginator>
+          <NePaginator
+            :current-page="currentPage"
+            :nav-pagination-label="t('ne_table.pagination')"
+            :next-label="t('ne_table.go_to_next_page')"
+            :page-size="pageSize"
+            :page-size-label="t('ne_table.show')"
+            :previous-label="t('ne_table.go_to_previous_page')"
+            :range-of-total-label="t('ne_table.of')"
+            :total-rows="rules.length"
+            @selectPageSize="(size: number) => { pageSize = size }"
+            @select-page="(page: number) => { currentPage = page }"
+          />
+        </template>
+      </NeTable>
+    </template>
+    <NeEmptyState
+      v-else
+      :description="t('standalone.ips.no_disabled_rules_description')"
```

Comment:
IMHO we don't need a description on this empty state

Reply:
I agree, but it's there :grimacing: 
https://www.figma.com/design/FhVUA3F59rOsQPh2bDlZ7V/NethSecurity-shared-file?node-id=2464-102500&t=lsGXknmI8lgkLHZA-4